python/ray/data/tests/test_parquet.py

```diff
@@ -1204,6 +1205,92 @@ def test_partitioning_in_dataset_kwargs_raises_error(ray_start_regular_shared):
         )
 
 
+def test_tensors_in_tables_parquet(
```

Comment:
Any reason not to parameterize this test?

Reply:
No need -- this test verifies BWC